CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# dbt_microsoft_ads_source v0.9.0
+
+[PR #31](https://github.com/fivetran/dbt_microsoft_ads_source/pull/31) includes the following updates:
+
+## Bug Fixes
+- Accommodates the recent addition of the `budget_name` and `budget_status` fields to the `CAMPAIGN_PERFORMANCE_DAILY_REPORT` source table. These fields are now:
```

Comment:
I don't know if we should mention that these fields are recent, since they have technically been there for years. Instead can we do the following:
```suggestion
- Accommodates when the `budget_name` and `budget_status` fields are populated within the `CAMPAIGN_PERFORMANCE_DAILY_REPORT` source table. These fields are now:
```